site-src/index.md

```diff
@@ -1,4 +1,4 @@
-# Introduction
+## Introduction
```

Comment:
I wonder why move everything one level down?

Reply:
The Table of Contents on the right side of the page was ignoring any singular `#` sections. I thought that was a bit confusing for new readers to the page. Moving each one a level down fixes this

<img width="327" alt="image" src="https://github.com/user-attachments/assets/ec5f08ed-a6e8-439b-8a4c-9b8a5097f391" />

Reply:
I think it’s better to fix the configuration (#31) instead of manually shifting everything down a level.

Reply:
Yup, that's much better. Closing this

Reply:
Could you keep the added KEP links?

Reply:
Sure! I'll reopen with that

Reply:
https://github.com/kubernetes-sigs/sig-multicluster-site/pull/35